src/main/java/com/andrew/strings/StringUtilsImpl.java

```diff
@@ -0,0 +1,88 @@
+package com.andrew.strings;
+
+import java.util.ArrayList;
+
+public class StringUtilsImpl implements StringUtils {
+    @Override
+    public int charCount(String str, char letter) {
```

Comment:
not sure that imlementation is correct? Example:

```
charCount("hello", 'h')
```

func returns me `0` in this case, doesn't it?